### PR TITLE
Remove color-brand-light and text-accent2-hover CSS colors

### PR DESCRIPTION
**Pull request by ranquild**
Related to https://github.com/metabase/metabase-private/issues/73

Refactors `color-brand-light` and `color-accent2` css colors to emotion. 

How to verify:
- For the native editor, make sure the autocomplete popup has same colors as before.

Screenshot 2023-07-17 at 16 28 43

### CODE REVIEW

File: frontend/src/metabase/components/Sidebar.css [hunk 1]
```diff
@@ -69,10 +69,3 @@
 :local(.icon) {
   composes: flex-no-shrink from "style";
 }
-
-:local(.noLabelsMessage) {
```
Comment (ranquild) [inline frontend/src/metabase/components/Sidebar.css:73]: Not used

File: frontend/src/metabase/css/core/colors.css [hunk 2]
```diff
@@ -43,21 +42,6 @@
...
-.text-brand-light-hover:hover,
-:local(.text-brand-light-hover):hover {
-  color: var(--color-brand-light);
-}
-
-.bg-brand-light {
-  background-color: var(--color-brand-light);
-}
-
-.bg-brand-dark {
```
Comment (ranquild) [inline frontend/src/metabase/css/core/colors.css:57]: Not used

File: frontend/src/metabase/query_builder/components/NativeQueryEditor.styled.tsx [hunk 3]
```diff
@@ -90,3 +91,34 @@ export const NativeQueryEditorRoot = styled.div`
     background-color: ${color("bg-light")};
   }
 `;
+
+export const aceEditorStyles = css`
```
Comment (ranquild) [inline frontend/src/metabase/query_builder/components/NativeQueryEditor.styled.tsx:95]: The popover is displayed outside of the element in the document.body; we cannot have these styles in `NativeQueryEditorRoot`

**Review by uladzimirdev — APPROVED** [review]
LGTM

**Pull request merged**